src/App.js

```diff
@@ -1,19 +1,128 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const todosFromServer = todos.map(todo => ({
+  ...todo,
+  name: users.find(user => user.id === todo.userId).name,
+}));
+
+class App extends React.Component {
+  state = {
+    people: users,
+    todoList: todosFromServer,
+    task: '',
+    userName: '',
+    id: todosFromServer[todosFromServer.length - 1].id + 1,
```

Comment:
redundant, let's do calculate it when setting state or just use Date.now()

Reply:
It is task condition https://prnt.sc/xr69p1